app/src/org/commcare/activities/InstallFromListActivity.java

```diff
@@ -363,9 +361,13 @@ public void processOther(int responseCode) {
     }
 
     @Override
-    public void handleIOException(IOException exception) {
-        Logger.log(LogTypes.TYPE_ERROR_SERVER_COMMS,
-                "An IOException was encountered during get available apps request: " + exception.getMessage());
+    public void handleException(Exception exception) {
```

Comment:
This explains the fuzzing of the exception type earlier. I think it would be better to type the plaintext password exception as an IOEXception that we specifically catch and log rather than moving the processing to take generic exceptions. (We already do similar things with HTTPS bad certificate responses).